### PR TITLE
[12.0] [MIG] account_payment_purchase

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Account Payment Purchase
========================
This module adds 2 fields on purchase orders: *Bank Account* and *Payment Mode*. These fields are copied from partner to purchase order and then from purchase order to supplier invoice.

This module is similar to the *purchase_payment* module; the main difference is that it doesn't depend on the *account_payment_extension* module (it's not the only module to conflict with *account_payment_extension*; all the SEPA modules in the banking addons conflict with *account_payment_extension*).

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please sign the CLA and check runbot and Travis status.

**Comment by Nikul-Chaudhary** [comment]
> Please sign the CLA and check runbot and Travis status.

@pedrobaeza I already sent mail `cla@odoo-community.org & info@odoo-community.org`

Can you please check and Process for that. Thanks.


screenshot from 2019-01-20 10-50-15

File: account_payment_purchase/__manifest__.py [hunk 1]
```diff
@@ -14,7 +14,8 @@
     'website': 'https://github.com/OCA/bank-payment',
     'depends': [
         'purchase',
-        'account_payment_partner'
+        'account_payment_partner',
+        'stock',
```
Comment (pedrobaeza) [inline account_payment_purchase/__manifest__.py]: You have to depend from `purchase_stock`, as you are using data from it.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Done!

**Review by pedrobaeza — APPROVED** [review]

**Comment by Nikul-Chaudhary** [comment]
@OCA/banking-maintainers Could you please review it?

**Review by MiquelRForgeFlow — APPROVED** [review]
code review LGTM

**Pull request merged**